cmd/geth/main.go

```diff
@@ -417,6 +428,35 @@ func attach(ctx *cli.Context) {
 	}
 }
 
+func initGenesis(ctx *cli.Context) {
+	genesisPath := ctx.Args().First()
+	if len(genesisPath) == 0 {
+		utils.Fatalf("must supply path to genesis JSON file")
+	}
+
+	var (
+		datadir = utils.MustMakeDataDir(ctx)
+		cache   = ctx.GlobalInt(utils.CacheFlag.Name)
+		handles = utils.MakeDatabaseHandles()
```

Comment:
Ah, Indeed. Honestly I just copy pasted it from the old code :-) thank you for the suggestion.